starter-code/pizza.js

```diff
@@ -1 +1,103 @@
 // Write your Pizza Builder JavaScript in this file.
+
+// $(document).ready(function() {
+
+// BUTTON VARS
+
+var btnPepperoni = $('.btn-pepperonni');
+
+var btnMushrooms = $('.btn-mushrooms');
+
+var btnGreenPeppers = $('.btn-green-peppers');
+
+var btnSauce = $('.btn-sauce');
+
+var btnCrust = $('.btn-crust');
+
+var btnWhite = $('.btn-sauce');
+
+var totalPriceDiv = $('strong');
+var actualTotalPrice = totalPriceDiv.html();
+
+var priceList = $('.price li');
+
+function getTotalPrice() {
+    var priceList = $('.price li').not(':hidden');
+    var grandTotal = 0;
+
+    $('.price li').each(function( index ) {
+        grandTotal += Number((priceList.eq(index).text().charAt(1)));
+      });
+
+    actualTotalPrice = grandTotal + 10;
+
+$(totalPriceDiv).text(`$${actualTotalPrice}`);
+
+    return actualTotalPrice;
+}
+
+// START WITH GLUTEN TOGGLED OFF
+
+btnCrust.removeClass('active');
+
+// CLICK FUNCTIONS FOR EACH BUTTON
+
+btnPepperoni.click(function(event) {
+        $('.pep').toggle();
+        btnPepperoni.toggleClass('active');
+        priceList.eq(0).toggle();
+        getTotalPrice();
+});
+
+btnMushrooms.click(function(event) {
+    $('.mushroom').toggle();
+    btnMushrooms.toggleClass('active');
+    priceList.eq(1).toggle();
+    getTotalPrice();
+});
+
+btnGreenPeppers.click(function(event) {
+    $('.green-pepper').toggle();
+    btnGreenPeppers.toggleClass('active');
+    priceList.eq(2).toggle();
+    getTotalPrice();
+});
+
+btnWhite.click(function(event) {
+    $('.sauce-white').toggle();
```

Comment:
using toggle tends to toggle all classes. To have it target just the class you want, you can do this `$('.sauce').toggleClass('sauce-white');`